ios/ReactTestApp/ContentView.swift

```diff
@@ -49,8 +49,8 @@ final class ContentViewController: UITableViewController {
 
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
             self?.reactInstance.initReact { components in
-                let items: [NavigationLink] = components.map { (name, component) in
-                    (component.displayName ?? name, { self?.navigate(to: component, name: name) })
+                let items: [NavigationLink] = components.map { (component) in
+                    (component.displayName ?? component.appKey, { self?.navigate(to: component) })
```

Comment:
I am not sure if I understand this expression correctly, so feel free to dismiss this comment if I am wrong. Shouldn't the `component.appKey` null-checked first and then `displayName`?

Reply:
`appKey` is what used to be the keys under `components`. It is required.